.github/workflows/optional-test.yaml

```diff
@@ -67,7 +67,7 @@ jobs:
             prisma-json-schema-generator,
             prisma-nestjs-graphql,
           ]
-        engine: ['binary'] #['node-api', 'binary']
+        clientEngine: ['node-api'] #['node-api', 'binary']
```

Comment:
Why the switch / only testing node-api now?

Reply:
This PR switches the default engine type from `binary` to `node-api`, so all the tests that only test one of these (for capacity reasons), also is switched over.

Reply:
Changed back to `['binary']`